documentation/source/user_section/installation.rst

```diff
@@ -80,9 +80,78 @@ recommend you download the Github version.
 Install on Windows 10 with WSL
 ------------------------------
 
-Windows subsystem for Linux (WSL) is available on Windows 10 and it makes it possible to run native Linux programs,
-such as SCT. Checkout the `installation tutorial for WSL <https://github.com/neuropoly/spinalcordtoolbox/wiki/SCT-on-Windows-10:-Installation-instruction-for-SCT-on-Windows-subsytem-for-linux>`_.
+Windows subsystem for Linux (WSL) is available on Windows 10 and it makes it possible to run native Linux programs, such as SCT.
 
+#. Install Windows Subsystem for Linux (WSL)
+
+   - Install `Xming <https://sourceforge.net/projects/xming/>`_.
+
+   - Install  `Windows subsystem for linux and initialize it <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`_.
+
+       - Please install the Ubuntu 18.04 LTS distro.
+
+#. Environment preparation
+
+   Run the following command to install various packages that will be needed to install FSL and SCT. This will require your password
+
+   .. code-block:: sh
+
+      sudo apt-get -y update
+      sudo apt-get -y install gcc
+      sudo apt-get -y install unzip
+      sudo apt-get install -y python-pip python
+      sudo apt-get install -y psmisc net-tools
+      sudo apt-get install -y git
+      sudo apt-get install -y gfortran
+      sudo apt-get install -y libjpeg-dev
+      echo 'export DISPLAY=127.0.0.1:0.0' >> .profile
+
+#. Install SCT
+
+   Download SCT:
+
+   .. code-block:: sh
+
+      git clone https://github.com/neuropoly/spinalcordtoolbox.git sct
+      cd sct
+
+   To select a `specific release <https://github.com/neuropoly/spinalcordtoolbox/releases>`_, replace X.Y.Z below with the proper release number. If you prefer to use the development version, you can skip this step.
+
+   .. code-block:: sh
+
+      git checkout X.Y.Z
+
+   Install SCT:
+
+   .. code:: sh
+
+      yes | ./install_sct
+
+#. Install FSL
+
+   FSL contains fsleyes which is the default viewer for NIfTI image in SCT.
```

Comment:
is FSLeyes still part of the whole FSL package? and by the way: now that FSLeyes is a standalone package, shouldn't we recommend instead of install FSLeyes (much lighter)?

Reply:
Ah, good observation! From [the FslWiki:](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FSLeyes#Install_from_conda-forge_.28recommended.29)

> FSLeyes can either be installed as part of FSL, installed into a conda environment, or installed into a Python environment (for advanced users).

Since we already use a conda environment for the SCT install, this does indeed seem like a much simpler way of going about it.

Reply:
@lrouhier @gaspardcereza can you confirm installation of FSLeyes within WSL is seamless?

Reply:
I didn't try to run FSLeyes yet (WSL2 is not really mean to run GUIs...). I think there's a way to do so but the few tutorials I saw were very advanced stuff (at least to me).  I can only say that installing FSL following their [instructions](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FslInstallation/Linux) went well (even if it required ~1h on my end) and that prelude is working. Maybe @lrouhier managed to launch FSL eyes on his end (if so that would be of interest to me because I don't really have a good nifti viewer).

Reply:
Last time it wasn't working really well because I had some dependency issues.. I can try again if need be. 
Fsleyes works really well once you installed (At least it is still working on my end). I am still using WSL 1 however (an old discussion about the why: https://github.com/neuropoly/spinalcordtoolbox/issues/2747)

Reply:
thank you for your feedback @gaspardcereza and @lrouhier 

@lrouhier :
> Fsleyes works really well once you installed 

i'm not sure i understand what you meant: "Fsleyes works really well once you installed..." once you installed what? FSL? or FSLeyes with the proper dependencies?

Reply:
> i'm not sure i understand what you meant: "Fsleyes works really well once you installed..." once you installed what? FSL? or FSLeyes with the proper dependencies?

Sorry Yes I meant "once you installed fsl".

Reply:
ok, so do you suggest that we ask our users to install FSL in order to use FSLeyes, because it was easier than installing FSLeyes as standalone?

Reply:
Yes I think so.

Reply:
Hi! Maybe that can help you.
I use the standalone FSLeyes in a conda environment in WSL1. I installed it and the wxPython dependancy according to ADS documentation [here](https://axondeepseg.readthedocs.io/en/latest/documentation.html#linux-tested-on-ubuntu).
For the display, I use vcXsrv (similar to Xming).
I never installed the full FSL, so I don’t know which one is easier. For me, the FSLeyes standalone installation is working well and was easy and fast to install.

Reply:
Thank you for your feedback @mariehbourget! If you found an easy solution to install FSLeyes within WSL, then we should definitely replace the current approach with your suggestion. Do you mind modifying the current SOP with your approach? [here](https://github.com/neuropoly/spinalcordtoolbox/pull/2952/files). Thank you so much!

Reply:
That is interesting as the newest conda version might work great (I tested it back in March). It might have been a wxpython install as mentionned in [your link](https://axondeepseg.readthedocs.io/en/latest/documentation.html#known-issues). That's why I went with the fsl install.

Reply:
I don't have permissions to modify directly in this PR, should I use a fork?

Reply:
@alexfoias or @joshuacwnewton can you add Marie Helene please? thanks

Reply:
@jcohenadad I added @mariehbourget on the repo with WR.